ksqldb-functional-tests/src/test/resources/query-validation-tests/select.json

```diff
@@ -6,17 +6,17 @@
     {
       "name": "new query planner test",
       "statements": [
-        "CREATE STREAM INPUT (id INT KEY, name STRING, foo STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
```

Comment:
Would it be better to add new test cases to show features of the new query planner? 

(If the general strategy is to update this one test as the new query planner evolves, that's fine as well.)